packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+// eslint-disable-next-line no-restricted-imports
+import * as Ariakit from '@ariakit/react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS } from '../utils';
+import { space } from '../utils/space';
+import type { CustomSelectProps } from './types';
+
+export const CustomSelectLabel = styled( Ariakit.SelectLabel )`
+	font-size: 11px;
+	font-weight: 500;
+	line-height: 1.4;
+	text-transform: uppercase;
+	margin-bottom: ${ space( 2 ) };
+`;
+
+const inputHeights = {
+	default: 40,
+	small: 24,
+};
+
+const getVariableInputStyles = (
+	size: NonNullable< CustomSelectProps[ 'size' ] >,
+	hasCustomRenderProp: boolean
+) => {
+	return css`
+		${ hasCustomRenderProp ? 'min-height' : 'height' }: ${ inputHeights[
+			size
+		] }px;
+		font-size: ${ size === 'small' && ! hasCustomRenderProp
```

Comment:
`${ size === 'small' && ! hasCustomRenderProp` could be set to a variable and reused for brevity.